assets/scss/00-base/_colors.scss

```diff
@@ -1,58 +1,8 @@
-// Theme colors
+// Mayflower Color Theme and Usage Mapping
 
-// Primary Color
-$c-bay-blue:                       #14558f;
-$c-bay-blue-lighter:               #8AAAC7;
-$c-bay-blue-lightest:              #E7EEF4;
+@import '../../colors/mayflower-color-tokens';

```

Comment:
Maybe a longer term change if it's not easy to adopt right now: [Sass is eventually deprecating `@import` in favor of the newer `@use` and `@forward` syntax](https://sass-lang.com/blog/the-module-system-is-launched). This has some benefits, like allowing consuming projects to control the namespace for imported resources. For example:

```
@use "mayflower-color-tokens" as mayflower;

$uswds-blue: mayflower.$c-bay-blue

```

Reply:
good to know! thanks! I don't think it's too much to do for just the colors, I will create a follow-up PR to use `@use`